code/modules/unit_tests/deletions.dm

```diff
@@ -0,0 +1,7 @@
+/// This is for regression tests of deletions that used to runtime.
+/// This would ideally be replaced by Del The World, unit testing every single deletion.
+/datum/unit_test/deletion_regressions
+
+/datum/unit_test/deletion_regressions/Run()
```

Comment:
hehehe what

Reply:
[![That way we have it!](https://img.youtube.com/vi/dL8UXQ6BIrc/0.jpg)](https://www.youtube.com/watch?v=dL8UXQ6BIrc )